src/tests/ftest/pool/query_attribute.py

```diff
@@ -54,17 +54,18 @@ def test_query_attr(self):
         query_result = daos_cmd.pool_query(pool=self.pool.uuid)
         actual_uuid = query_result["response"]["uuid"]
         actual_size = query_result["response"]["tier_stats"][0]["total"]
+        actual_size_roundup = int(actual_size/100000)*100000
 
         expected_uuid = self.pool.uuid.lower()
         if expected_uuid != actual_uuid:
             msg = "Unexpected UUID from daos pool query! " +\
                 "Expected = {}; Actual = {}".format(expected_uuid, actual_uuid)
             errors.append(msg)
 
-        if expected_size != actual_size:
-            msg = "Unexpected Total Storage Tier 0 size from daos pool " +\
-                "query! Expected = {}; Actual = {}".format(
-                    expected_size, actual_size)
+        if expected_size != actual_size_roundup:
+            msg = "#Unexpected Total Storage Tier 0 size from daos pool " +\
```

Comment:
Is # a typo? "#Unexpected...'

Reply:
Thanks for the review.  this is not a typo, add # for easier debug in case of failure.